src/nvim/window.c

```diff
@@ -5717,7 +5717,7 @@ int match_delete(win_T *wp, int id, int perr)
   }
   if (cur == NULL) {
     if (perr == TRUE)
```

Comment:
Note that here and above it makes sense to delete `== TRUE` completely rather then lowercase it.